nixos/modules/services/misc/paperless.nix

```diff
@@ -380,6 +390,27 @@ in
         services.paperless.manage = manage;
         environment.systemPackages = [ manage ];
 
+        services.nginx = lib.mkIf cfg.configureNginx {
+          enable = true;
+          upstreams.paperless.servers."${cfg.address}:${toString cfg.port}" = { };
+          virtualHosts.${cfg.domain} = {
+            forceSSL = lib.mkDefault true;
+            locations = {
+              "/".proxyPass = "http://paperless";
+              "/static/" = {
+                root = config.services.paperless.package;
+                extraConfig = ''
+                  rewrite ^/(.*)$ /lib/paperless-ngx/$1 break;
```

Comment:
This should probably be `dataDir`.

Reply:
The rewrite logic is a bit weird here but I tested this in depth a while ago.